content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -95,42 +86,52 @@ There may be cases when a workspace contains conflicting variables of the same t
 
 Terraform Cloud prioritizes and overwrites conflicting variables according to the following precedence:
 
-### 1. Workspace-Specific Variables
+### 1. Command line Argument Variables
```

Comment:
I don't want to get "Variables Set" mixed up with "Variable Sets" -- also via is a latin word, right?

Reply:
We use via in other places throughout the docs, but that's a fair point. I also see your point about leaving this as is so that two things don't start with "Variables" - thank you!